Blogger.Sample/Blogger.Sample.Windows/ViewModel/MainViewModel.cs

```diff
@@ -0,0 +1,34 @@
+using GalaSoft.MvvmLight;
```

Comment:
Why do we have a change here?
Is it because you upgraded from MvvmLightLibs.4.3.31.1 to CommonServiceLocator.1.3?